docs/white_papers.toml

```diff
@@ -16,3 +16,14 @@ pdf_url = "https://github.com/felix-russo/gaussian-track-densities/releases/down
 authors = [ "Felix Russo",]
 title = "Gaussian Track Densities"
 description = "Given the impact parameters $(d_0, z_0, t_0)$ of a track in Perigee parametrization, one can model the probability of the particle passing exactly through a point $(d, z, t)$ using a multivariate Gaussian distribution. In this white paper, we derive the maximum and the width of such a distribution for $d = 0$. This is useful in vertex seed finding, where we only consider the track density along the beam axis. We use the analytical results from this white paper in the unit test of the Acts module AdaptiveGridTrackDensity."
+
+[[white_papers]]
+repository = "https://github.com/beomki-yeo/path-correction-term-from-momentum-variation"
+slug = "correction-for-transport-jacobian"
+pdf_url = "https://github.com/beomki-yeo/path-correction-term-from-momentum-variation/releases/download/v1.0/main.pdf"
+
+[white_papers.metadata]
+authors = [ "Beomki Yeo",]
+title = "A correction term for transport jacobian from the direction variation"
```

Comment:
nitpick: The title should be capitalized correctly, see https://capitalizemytitle.com/ for example :)

Reply:
Which style are you exactly following? Not all journal publications have their titles in captilized style

Reply:
Capitalization is done anyway